src/adapter/src/metrics.rs

```diff
@@ -120,6 +121,12 @@ impl Metrics {
                 help: "Number of optimization notices per notice type.",
                 var_labels: ["notice_type"],
             )),
+            append_table_seconds: registry.register(metric!(
+                name: "mz_append_table_seconds",
```

Comment:
```suggestion
                name: "mz_append_table_duration_seconds",
```

Prometheus seems to recommend using duration for this stuff: https://prometheus.io/docs/practices/naming/#metric-names

Reply:
Sweet, renamed!